modules/ossm-cr-version.adoc

```diff
@@ -5,4 +5,4 @@
 [id="ossm-cr-version_{context}"]
 = version parameter
 
-You use the `version` parameter to specify what version of the control plane to install. When you create a `ServiceMeshControlPlane` object with an empty `version` parameter, the admission webhook sets the version to the current version. New `ServiceMeshControlPlanes` objects with an empty `version` parameter are set to `v2.0`. Existing `ServiceMeshControlPlanes` objects with an empty `version` parameter keep their setting.
+You use the `version` parameter to specify what version of the control plane to install. When you create a `ServiceMeshControlPlane` object with an empty `version` parameter, the admission webhook sets the version to the current version. If you do not specify a `version` parameter when creating your SMCP, the Operator will apply the latest version ({MaistraVersion}). Existing `ServiceMeshControlPlanes` objects with an empty `version` parameter keep their setting.
```

Comment:
What about something like this:

The OSSM operator supports installation of different versions of a control plane. Use the `version` parameter to specify what version of the control plane to install. If you do not specify a `version` parameter when creating your SMCP, the Operator will apply the latest version: ({MaistraVersion}). Existing `ServiceMeshControlPlanes` objects with an empty `version` parameter keep their setting.